library/src/main/java/org/isaacphysics/graphchecker/features/PointsFeature.java

```diff
@@ -71,27 +72,31 @@ public boolean test(Line line) {
                 return false;
             }
 
-            return Streams.zip(expectedPoints.stream(), line.getPointsOfInterest().stream(),
-                (expected, actual) -> expected.getLeft() == actual.getPointType()
-                    && settings().getSectorClassifier().classifyAll(actual).contains(expected.getRight()))
-                .allMatch(Boolean::booleanValue);
+            return Streams.zip(expectedPoints.stream(), line.getPointsOfInterest().stream(), this::pointsMatch)
+                    .allMatch(Boolean::booleanValue);
+        }
+        
+        protected boolean pointsMatch(ImmutablePair<PointType, Sector> expected, PointOfInterest actual) {
+            return expected.getLeft() == actual.getPointType() 
+                    && expected.getRight() == settings().getSectorBuilder().byName(SectorBuilder.ANY) 
+                        || (settings().getSectorClassifier().classifyAll(actual).contains(expected.getRight()));
```

Comment:
I think `&&` binds tighter than `||`, so just checking that this shouldn't be:
```suggestion
            return expected.getLeft() == actual.getPointType() 
                    && (expected.getRight() == settings().getSectorBuilder().byName(SectorBuilder.ANY) 
                        || settings().getSectorClassifier().classifyAll(actual).contains(expected.getRight()));
```

Reply:
Thanks for spotting that, fixed!